tests/main/views/test_drafts.py

```diff
@@ -362,10 +407,13 @@ def post_to_copy_published_from_framework(
             "supplierId": 1,
         }
 
-        if override_questions_to_copy == []:
+        if override_questions_to_copy == [] or override_questions_to_exclude == []:
             pass
-        elif override_questions_to_copy:
-            data.update(questionsToCopy=override_questions_to_copy)
+        elif override_questions_to_copy or override_questions_to_exclude:
+            if override_questions_to_copy:
+                data.update(questionsToCopy=override_questions_to_copy)
+            if override_questions_to_exclude:
+                data.update(questionsToExclude=override_questions_to_exclude)
         else:
             data.update(self.basic_questions_json)
```

Comment:
I found this logic quite hard to untangle.

I think what is happening is that we include `questionsToCopy` (from `self.basic_questions.json`) in the request unless either `override_questions_to_copy` or `override_questions_to_exclude` is specified.

Is that what we want?

Reply:
For now, yes. This test setup a bit of a mess really. I considered refactoring, but didn't want to yank the bit of string and unravel the whole jumper 😹